internal/operands/common-templates/resource.go

```diff
@@ -0,0 +1,129 @@
+package common_templates
+
+import (
+	"bytes"
+	templatev1 "github.com/openshift/api/template/v1"
+	"io"
+	"io/ioutil"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"sync"
+)
+
+const (
+	goldenImagesNSname = "kubevirt-os-images"
+	version            = "v0.12.2"
```

Comment:
Maybe instead of hard-coding the version, we should select the latest version found in that directory?

Reply:
or have it in CR config?

Reply:
We can have it in the CR config, but it should default to the latest version when unspecified